### PR TITLE
Remedy for the import grouping I broke

### PR DESCRIPTION
**Pull request by Telzhaak**
My PR #974 broke the import grouping, so I've begun to fix them again, while also `nesting` the use statements.

Before I give this treatment to every file, I'd like to know how this looks, whether something should be done differently, whether or not to nest `lib.rs` `pub use` statements, etc

- [x] animations
- [x] assets
- [x] audio
- [x] config
- [x] controls
- [x] core
- [x] gltf
- [x] input
- [x] locale
- [x] network
- [x] renderer
- [x] ui
- [x] utils
- [x] amethyst

### CODE REVIEW

File: amethyst_animation/src/bundle.rs [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+use std::{hash::Hash, marker};
+
 use resources::AnimationSampling;
 use skinning::VertexSkinningSystem;
-use std::hash::Hash;
-use std::marker;
+
+use amethyst_core::{
+    specs::prelude::{Component, DispatcherBuilder},
+    {Result, SystemBundle},
```
Comment (torkleyy) [inline amethyst_animation/src/bundle.rs]: No need to group those (also doesn't look very good).

Comment (Telzhaak) [reply amethyst_animation/src/bundle.rs]: ah yeah, forgot to remove the `{}` around Result and SystemBundle.
Other than that, why not group those?

Comment (Rhuagh) [reply amethyst_animation/src/bundle.rs]: Because they are part of the outer grouping

**Comment by torkleyy** [comment]
Removing the braces is all I wanted to ask for ;)

File: amethyst_animation/src/bundle.rs (hunk 1 above)
Comment (Telzhaak) [reply amethyst_animation/src/bundle.rs]: So:
```rust
use amethyst_core::specs::prelude::{Component, DispatcherBuilder};
use amethyst_core::{Result, SystemBundle};
```?

Comment (torkleyy) [reply amethyst_animation/src/bundle.rs]:
```rust
use amethyst_core::{
    specs::prelude::{Component, DispatcherBuilder},
    Result,
    SystemBundle,
};
```

Very simple, sorry for being unclear.

**Comment by torkleyy** [comment]
@Telzhaak Are you still working on this PR?

**Comment by Telzhaak** [comment]
Yes, it isn't my highest priority though

**Comment by Telzhaak** [comment]
@torkleyy If you want, you can make a half-way review

**Comment by Telzhaak** [comment]
- [x] rustfmt 0.99.4-stable (1c408818 2018-08-27)
- [x] cargo test --all
- [x] cargo test --all --release
- [x] cargo +nightly test --all --features "nightly"
- [x] cargo +nightly test --all --release --features "nightly"

**Comment by torkleyy** [comment]
Let's merge (and break all the other PRs (sorry)).

bors r+

**Comment by torkleyy** [comment]
Okay, bors is not happy..

**Comment by torkleyy** [comment]
Here you go
bors r+

**Comment by Telzhaak** [comment]
Those poor souls that have to rebase now,
I feel their pain

**Comment by jojolepro** [comment]
>Those poor souls that have to rebase now,
I feel their pain

Thanks, rip my open pr that was waiting for reviews and the ui one I'm working on ^^

Especially the ui one, it has a TON of `use` moving

**Pull request merged**